vllm/spec_decode/spec_decode_worker.py

```diff
@@ -366,7 +385,7 @@ def _run_non_driver_rank(self) -> bool:
         for _ in range(max(num_lookahead_slots, 1)):
             self.proposer_worker.execute_model()
 
-        self.scorer_worker.execute_model()
+        self.scorer_worker.execute_model()  #TODO hidden states tbd
```

Comment:
What does the comment mean here?

Reply:
Oh just that we still need to fix TP, hopefully today.